tests/foreman/ui/test_discoveredhost.py

```diff
@@ -1260,3 +1405,60 @@ def test_positive_rebuild_dns_on_provisioning(self):
 
         @CaseLevel: System
         """
+
+
+class DiscoveryPrefixTestCase(UITestCase):
+    """Test around updating Discovery Prefix"""
+
+    @classmethod
+    def setUpClass(cls):
+        super(DiscoveryPrefixTestCase, cls).setUpClass()
+        cls.org = entities.Organization(name=gen_string('alpha')).create()
+        cls.org_name = cls.org.name
+        # Update hostname_prefix with some string other than default 'mac'
+        cls.prefix = 'dhost'
+        cls.discovery_prefix = entities.Setting().search(
+            query={'search': 'name="discovery_prefix"'})[0]
+        cls.default_prefix = str(cls.discovery_prefix.value)
+        cls.discovery_prefix.value = cls.prefix
+        cls.discovery_prefix.update({'value'})
```

Comment:
Set? Isn't that a typo? I wasn't even aware we support sets for update, we usually use lists like this: `cls.discovery_prefix.update(['value'])`. But if you're sure it's working - i'm ok with it.